modules/iopcore/cdvdman/device-mmce.c

```
@@ -166,24 +166,34 @@ int DeviceReadSectors(u64 lsn, void *buffer, unsigned int sectors)
 //TODO: For VMCs
 int mmce_read_offset(int fd, unsigned int offset, unsigned int size, unsigned char *buffer)
```

Cppcheck (2.21.0)

[style] The function 'mmce_read_offset' is never used.

(unusedFunction)